src/Stryker.Core/Stryker.Core/Reporters/Json/SourceFiles/JsonMutant.cs

```diff
@@ -38,6 +38,8 @@ public JsonMutant(IReadOnlyMutant mutant)
             StatusReason = mutant.ResultStatusReason;
 
             Static = mutant.IsStaticValue;
+
+            CoveredBy = mutant.CoveringTests.GetGuids()?.Select(g => g.ToString());
```

Comment:
So where is this going to be displayed? If in the final report, I'd prefer having a test name these instead of the Id.

Reply:
This is used to find the covering test from the list of tests in generating the visual report. How the report then decides to display this lives in the code for the report and should be changed there if the current view is not optimal.

Reply:
Cool.